open-company/open-company-web
# Remove thumbs-up control from comments

Trello card: https://trello.com/c/fMhFlJZn/2002-remove-the-thumbs-up-icon

### QA steps:

- Create a new post and add some comments to it from various users. Ensure that no thumbs up icon is present on any of them.

### Open questions

I'm unsure whether [comments-reaction-toggle](https://github.com/open-company/open-company-web/blob/remove-thumbs-up/src/oc/web/actions/comment.cljs#L76-L90) action should be removed or not. We will eventually introduce emojis, and so this action could potentially be retrofitted to handle that use case. **Please advise.**
